core/src/main/resources/hudson/tasks/ArtifactArchiver/help-followSymlinks.html

```diff
@@ -0,0 +1,6 @@
+<div>
```

Comment:
I recommend simplifying this. I don't think the user really needs to know about Ant and Directory Scanner. I would probably remove the help text entirely, as I think "Follow symbolic links" is sufficiently explanatory.